### PR TITLE
Feature/bsc 938427

### PR DESCRIPTION
**Pull request by taroth21**
@krig: Please review the changes (esp. xml/ha_hb_report.xml → xml/ha_crmreport_passl.xml). What I forgot to ask is how to change the following line in this file:
Cmnd_Alias HA_ALLOWED = /bin/su,/usr/sbin/hb_report,/usr/sbin/crm_report

With what do I need to replace '/usr/sbin/hb_report,/usr/sbin/crm_report' with?

### CODE REVIEW

File: xml/ha_crmreport_passl.xml [hunk 1]
```diff
@@ -363,8 +355,7 @@ Runas_Alias R = root <co xml:id="ha.sudoers.runas.alias"/></screen>
        This is useful if you want to limit what the non-root user can access
        when using <command>sudo</command>. In this case the
        <systemitem class="username">hareport</systemitem>
-       user account will need access to the commands
-       <command>hb_report</command> (or <command>crm_report</command>) and
+       user account will need access to the commands <command>crm&nbsp;report</command>
```
Comment (krig) [inline xml/ha_crmreport_passl.xml]: I suppose here it ought to be "command", not "commands"

File: xml/ha_crmreport_passl.xml [hunk 2]
```diff
@@ -336,7 +328,7 @@
     </para>
 <screen>Host_Alias	CLUSTER = &node1;,&node2;,&node3; <co xml:id="ha.sudoers.host.alias"/>
 User_Alias HA = hareport <co xml:id="ha.sudoers.user.alias"/>
-Cmnd_Alias HA_ALLOWED = /bin/su ,/usr/sbin/hb_report,/usr/sbin/crm_report <co xml:id="ha.sudoers.cmd.alias"/>
+Cmnd_Alias HA_ALLOWED = /bin/su,/usr/sbin/hb_report,/usr/sbin/crm_report <co xml:id="ha.sudoers.cmd.alias"/>
```
Comment (krig) [inline xml/ha_crmreport_passl.xml]: Hmm, I think this then ought to be 

`Cmnd_Alias HA_ALLOWED = /bin/su ,/usr/sbin/crm report *`

Comment (krig) [inline xml/ha_crmreport_passl.xml]: Hmm, I think this then ought to be 

`Cmnd_Alias HA_ALLOWED = /bin/su ,/usr/sbin/crm report *`

File: xml/ha_crmreport_passl.xml (hunk 1 above)
Comment (taroth21) [inline xml/ha_crmreport_passl.xml]: Thanks, fixed!

File: xml/ha_crmreport_passl.xml (hunk 2 above)
Comment (taroth21) [inline xml/ha_crmreport_passl.xml]: Thanks, fixed!

**Pull request closed without merging**